server/localize.js

```diff
@@ -40,7 +40,8 @@ module.exports = function localize(server, options) {
     if(knownLocales.indexOf(urlLocale) !== -1) {
       localizedUrl = orgUrl.replace(urlLocale, bestLanguage);
     } else {
-      localizedUrl = "/" + bestLanguage + "/" + (orgUrl === "/" ? "" : orgUrl);
+      // This regex makes sure that orgUrl always begins with `/` and inserts one if it doesn't
+      localizedUrl = "/" + bestLanguage + orgUrl.replace(/^\/?/, "/");
```

Comment:
possible nit: I don't know if line 38 is normalized or not. If you request `//en-US` for instance we want to end up with a single `/` so if it's nor normalized we probably want `^\/*` rather than `^\/?`